client/options.go

```diff
@@ -0,0 +1,352 @@
+package client
+
+import (
+	"dubbo.apache.org/dubbo-go/v3/common/constant"
+	"fmt"
+	"github.com/go-playground/validator/v10"
+	perrors "github.com/pkg/errors"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+type CallOptions struct {
+	RequestTimeout string
+	Retries        string
+}
```

Comment:
The options exposed here should be further investigated to make sure it's a 100% copy of the `config` package in regard of functionality.

Reply:
Yes, It's a placeholder that waits for professionals to think about what fields should be exposed.